### ISSUE TITLE
[Docs update] Fix codelists horizontal & vertical scroll, check hyperlinks

### ISSUE THREAD

**Issue by odscjen** [GFDRR/rdl-standard#161]
Check if there is a way to present the [open codelists](https://rdl-standard.readthedocs.io/en/dev/reference/codelists/#open-codelists) without having to horizontally scroll

**Comment by matamadio** [comment]
Please also consider introducing height limitation for huge tables such as media formats - with an internal scroll.
Also note in the schema reference (and possibly elsewhere) that the links to codelists #anchors with underscore should replace with dash:

E.g. https://rdl-standard.readthedocs.io/en/178-diagrams/reference/codelists/#function_approach does not work;
it is https://rdl-standard.readthedocs.io/en/178-diagrams/reference/codelists/#function-approach

**Comment by duncandewhurst** [comment]
All sounds good. I'll make those updates as part of https://github.com/GFDRR/rdl-standard/pull/214

**Comment by duncandewhurst** [comment]
> Best thing would be to have the hyperlink target to use underscore, to avoid confusion (since the schema codelists adopt underscore).

That isn't possible because Sphinx replaces the `_` character in hyperlink targets with `-` for compatibility with CSS. As such, I've updated the codelist URLs in the schema to use `-` in https://github.com/GFDRR/rdl-standard/pull/236.

There shouldn't be any other broken links because Sphinx checks for unresolvable references in Markdown content and we've updated the Sphinx directive that generates the schema tables to do the same replacements as Sphinx in https://github.com/OpenDataServices/sphinxcontrib-opendataservices-jsonschema/pull/51, but let me know if you find any.
